packages/material-ui/src/locale/index.js

```diff
@@ -983,14 +1063,18 @@ export const nlNL = {
 };
 
 export const plPL = {
+  localization: {
+    formatNumber: formatNumber('pl-PL'),
+  },
   props: {
     // MuiBreadcrumbs: {
     //   expandText: 'Show path',
```

Comment:
@eXtreme Thank you for the initial contribution. We have a couple of new keys, if you have some time, it would be awesome to translate them :).